rds-deployment/azuredeploy.json

```diff
@@ -102,24 +108,23 @@
     "lbBE": "ADLBBE",
     "publicBEAddressPoolID": "[concat(variables('publiclbID'),'/backendAddressPools/',variables('lbBE'))]",
     "gwLBName": "[concat('GWPLB',resourceGroup().name)]",
-    "gwIPAddressName": "[concat('GWPIP',resourceGroup().name)]",
-    "gwIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('gwIPAddressName'))]",
+    "publicIPAddressName": "[tolower(concat('adpip',resourceGroup().name))]",
+    "gwIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',parameters('gwpublicIPAddressName'))]",
     "gwlbFE": "GWLBFE",
     "gwlbID": "[resourceId('Microsoft.Network/loadBalancers',variables('gwLBName'))]",
     "gwlbFEConfigID": "[concat(variables('gwlbID'),'/frontendIPConfigurations/',variables('gwlbFE'))]",
     "gwlbBE": "GWLBBE",
     "gwBEAddressPoolID": "[concat(variables('gwlbID'),'/backendAddressPools/',variables('gwlbBE'))]",
-    "gwdnsLabelPrefix": "[concat('gwd', resourceGroup().name)]",
+    "dnsLabelPrefix": "[tolower(concat('adns', resourceGroup().name))]",
     "storageAccountName": "[tolower(concat('rdsa',uniqueString(resourceGroup().id)))]",
     "storageAccountType": "Standard_LRS",
-    "uniqueStorageAccountContainerName": "[concat('sc', resourceGroup().name)]",
+    "uniqueStorageAccountContainerName": "[tolower(concat('sc', resourceGroup().name))]",
```

Comment:
resourcegroup names can contain chars that are not valid for storage accounts, better practice would be to use the uniqueString() function